File: 2st/Yeonwoo/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    arr1_sum = 0
    arr2_sum = 0
    if len(arr1) != len(arr2):
        if len(arr1) > len(arr2):
            answer = 1
        elif len(arr1) < len(arr2):
            answer = -1
    elif len(arr1) == len(arr2):
        for i in arr1:
            arr1_sum += i
        for j in arr2:
            arr2_sum += j
        if arr1_sum > arr2_sum:
            answer = 1
        elif arr1_sum == arr2_sum:
            answer = 0
        elif arr1_sum < arr2_sum:
            answer = -1
    return answer

File: 3st/Juram/programmer/배열 비교하기.js
function solution(arr1, arr2) {
    let arr1Sum = 0;
    let arr2Sum = 0;
    if(arr1.length > arr2.length) {return 1;}
    else if(arr1.length == arr2.length) {
        for(let i=0; i<arr1.length; i++){
            arr1Sum += arr1[i];
        }
        for(let i=0; i<arr2.length; i++){
            arr2Sum += arr2[i];
        }
        if(arr1Sum > arr2Sum) return 1;
        else if(arr1Sum == arr2Sum) return 0;
        else return -1;
    }
    else {return -1;}
}